rholang/src/main/scala/coop/rchain/rholang/interpreter/Substitute.scala

```diff
@@ -19,7 +19,7 @@ trait Substitute[M[_], A] {
 }
 
 object Substitute {
-  def charge[A: Chargeable, M[_]: Sync: CostAccounting](
+  private[interpreter] def charge[A: Chargeable, M[_]: Sync: CostAccounting](
```

Comment:
I think making this private could be a separate commit :)

Reply:
Done